api/api.yaml

```diff
@@ -25,28 +25,34 @@ paths:
               properties:
                 providerWorkspaceRef:
                   type: string
-                  description: The identifier for the provider workspace
+                  description: The identifier for the provider workspace.
                   example: salesforce-instance-domain
                 projectId:
                   type: string
                   description: The Ampersand project ID.
+                  example: 453eac80-3482-47c5-b4d6-f040959499e3
                 groupRef:
                   type: string
                   description: The ID that your app uses to identify the group of users for this Connection, this can be an Org ID, workspace ID, etc.
+                  example: group-123
                 groupName:
                   type: string
                   description: The display name for the group. Defaults to groupRef if not provided.
+                  example: Organization Name
                 consumerRef:
                   type: string
                   description: The ID that your app uses to identify the user whose SaaS credential will be used for this OAuth flow.
+                  example: user_123456
                 consumerName:
                   type: string
                   description: The display name for the consumer. Defaults to consumerRef if not provided.
+                  example: John Doe
                 providerAppId:
                   type: string
                   description:
                     ID of the provider app (e.g. Salesforce connected app), returned from a CreateProviderApp call.
                     If omitted, the default provider app that was set up on the Ampersand Dashboard is assumed.
+                  example: ajVG9GCMQoQ6rpzQvXN4DwZT5NdFpOOSR2DMCrjKzC8F_XKh9tg8dLgv_xBenv4D5or2hMzUS2Z_b4VyTTnk
```

Comment:
@renukashiwakoti all the "IDs" are UUIDs, can you revise to look like a UUID? This is too long.